### ISSUE TITLE
Skip SSL verification not working

### ISSUE THREAD

**Issue by pschmitt** [terrycain/aiovault#11]
* aiovault version: 1.0.0
* Python version: 3.6.3
* Operating System: Arch Linux

### Description
The `verify` parameter seems to be ignored.

### What I Did

```
import aiovault
with aiovault.VaultClient(vault_url=addr, token=token, verify=False) as client:
        is_authed = await client.is_authenticated()
```

```
Traceback (most recent call last):
  File "/home/pschmitt/.local/share/virtualenvs/vaultrecur/lib/python3.6/site-packages/aiohttp/connector.py", line 788, in _wrap_create_connection
    return (yield from self._loop.create_connection(*args, **kwargs))
  File "/usr/lib64/python3.6/asyncio/base_events.py", line 803, in create_connection
    sock, protocol_factory, ssl, server_hostname)
  File "/usr/lib64/python3.6/asyncio/base_events.py", line 829, in _create_connection_transport
    yield from waiter
  File "/usr/lib64/python3.6/asyncio/sslproto.py", line 503, in data_received
    ssldata, appdata = self._sslpipe.feed_ssldata(data)
  File "/usr/lib64/python3.6/asyncio/sslproto.py", line 201, in feed_ssldata
    self._sslobj.do_handshake()
  File "/usr/lib64/python3.6/ssl.py", line 689, in do_handshake
    self._sslobj.do_handshake()
ssl.SSLError: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:777)

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "vaultrecur.py", line 37, in <module>
    main()
  File "vaultrecur.py", line 32, in main
    not args.insecure))
  File "/usr/lib64/python3.6/asyncio/base_events.py", line 467, in run_until_complete
    return future.result()
  File "vaultrecur.py", line 21, in list_all_entries
    is_authed = await client.is_authenticated()
  File "/home/pschmitt/.local/share/virtualenvs/vaultrecur/lib/python3.6/site-packages/aiovault/client.py", line 46, in is_authenticated
    await self._get('auth/token/lookup-self')
  File "/home/pschmitt/.local/share/virtualenvs/vaultrecur/lib/python3.6/site-packages/aiovault/base.py", line 82, in _get
    return await self._request('get', path, payload=None, params=params, wrap_ttl=wrap_ttl)
  File "/home/pschmitt/.local/share/virtualenvs/vaultrecur/lib/python3.6/site-packages/aiovault/base.py", line 169, in _request
    async with method_func as response:
  File "/home/pschmitt/.local/share/virtualenvs/vaultrecur/lib/python3.6/site-packages/aiohttp/client.py", line 692, in __aenter__
    self._resp = yield from self._coro
  File "/home/pschmitt/.local/share/virtualenvs/vaultrecur/lib/python3.6/site-packages/aiohttp/client.py", line 269, in _request
    conn = yield from self._connector.connect(req)
  File "/home/pschmitt/.local/share/virtualenvs/vaultrecur/lib/python3.6/site-packages/aiohttp/connector.py", line 393, in connect
    proto = yield from self._create_connection(req)
  File "/home/pschmitt/.local/share/virtualenvs/vaultrecur/lib/python3.6/site-packages/aiohttp/connector.py", line 740, in _create_connection
    _, proto = yield from self._create_direct_connection(req)
  File "/home/pschmitt/.local/share/virtualenvs/vaultrecur/lib/python3.6/site-packages/aiohttp/connector.py", line 851, in _create_direct_connection
    raise last_exc
  File "/home/pschmitt/.local/share/virtualenvs/vaultrecur/lib/python3.6/site-packages/aiohttp/connector.py", line 823, in _create_direct_connection
    req=req, client_error=client_error)
  File "/home/pschmitt/.local/share/virtualenvs/vaultrecur/lib/python3.6/site-packages/aiohttp/connector.py", line 793, in _wrap_create_connection
    raise ClientConnectorSSLError(req.connection_key, exc) from exc
aiohttp.client_exceptions.ClientConnectorSSLError: Cannot connect to host XXX-REDACTED-XXX ssl:True [[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:777)]
```

**NOTE**: I removed the server url from the traceback.

**Comment by terrycain** [comment]
Wow someone actually uses this other than me ;-). I'll have a look later

**Comment by terrycain** [comment]
Pretty sure I've fixed it in e761e8552554bf8d3c56659c5f6b754ca47d05ce. I set up the aiohttp session to respect verify=False, and then later on I completely ignore session I created.

Are you happy to test out a branch before I do a release? Worst case I can just push out a new version but if I can get the branch to go through a real world test that'll be better. (Branch [fix_11](https://github.com/terrycain/aiovault/tree/fix_11))

**Comment by pschmitt** [comment]
Yep, your fix works as intended. Thanks!

**Comment by terrycain** [comment]
Cool, should have that released soonish

**Comment by terrycain** [comment]
Released as `v1.1.0` :+1:
